dashboard/test/helpers/delete_accounts_helper_test.rb

```diff
@@ -955,6 +973,42 @@ class DeleteAccountsHelperTest < ActionView::TestCase
     end
   end
 
+  #
+  # Table: dashboard.featured_projects
+  #
+
+  test "unfeatures any featured projects owned by soft-deleted user" do
+    student = create :student
+    with_channel_for student do |channel_id|
+      featured_project = create :featured_project,
+        storage_app_id: channel_id,
+        featured_at: Time.now
+
+      assert featured_project.featured?
+
+      student.destroy
+
+      featured_project.reload
+      refute featured_project.featured?
+    end
+  end
+
+  test "unfeatures any featured projects owned by purged user" do
+    student = create :student
```

Comment:
Are users always soft deleted prior to being purged? If so, wouldn't their projects already be unfeatured? I don't think it hurts to be extra safe in this case, just curious how necessary this step is.

Reply:
Yes - if a user is not already soft-deleted, then soft-deleting is the first step when purging the account.  I thought it would be useful to keep this particular test in though - the implementation could conceivably change such that purged and soft-deleted became independent states, but we want this behavior for both of them.